src/ephemeris/shed_tools_methods.py

```diff
@@ -1,6 +1,19 @@
 from bioblend.toolshed import ToolShedInstance
 
 
+VALID_KEYS = [
+        "name",
+        "owner",
+        "changeset_revision",
+        "tool_panel_section_id",
+        "tool_panel_section_label",
+        "tool_panel_shed_url",
+        "install_repository_dependencies",
+        "install_resolver_dependencies",
+        "install_tool_dependencies"
```

Comment:
tool_shed_url is also valid, right ? I think I saw that in one of the failing tests